frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/employee/NewApplication/SuccessModal.js

```
@@ -0,0 +1,138 @@
+    <div style={styles.successModal}>
+      <div style={styles.successIcon}>
+        <span style={{ color: "white", fontSize: "1.5rem" }}>✔</span>
+      </div>
+      <h2 style={{ marginTop: "1rem" }}>{t("Application Submitted Successfully")}</h2>
+      <p style={{ color: "gray" }}>
+        {t("Application Number")}
+        <br />
+        {propertyId && <strong> {propertyId}</strong>}
+      </p>
+      <button onClick={onClose} style={styles.successButton}>
+        {t("Home")}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)